src/Control/Monad/Eff/JQuery.js

```diff
@@ -167,6 +167,20 @@ exports.setText = function(text) {
     };
 };
 
+exports.getHtml = function(ob) {
+    return function() {
+        return ob.html();
+    }
+}
+
+exports.setHtml = function(html) {
+    return function(ob) {
+        return function() {
+            return ob.html(html);
+        }
+    }
+}
```

Comment:
I hate to nitpick, but when we move this to core, we'll want to run it through a linter. Could you please add semicolons to every line, like for the other functions?